app/index.html

```diff
@@ -11,6 +11,6 @@
   <script src="./funding-hub.js"></script>
 </head>
 <body ng-app="fundingHubApp">
-  <h1><a ng-href="#/">FundingHub</a></h1>
+  <h1><a ng-href="#!/">FundingHub</a></h1>
```

Comment:
For some reason routing on Angular behaving differently but I think this is nothing to do with upgrading truffle to version 3